test/index.js

```diff
@@ -138,6 +138,11 @@ describe('creditCardType', function () {
       ['6221558812340000', 'unionpay'],
       ['6269992058134322', 'unionpay'],
       ['622018111111111111', 'unionpay'],
+      ['8100513433325374', 'unionpay'],
+      ['8111700872004845', 'unionpay'],
+      ['8141618644273338', 'unionpay'],
+      ['8158163233706018', 'unionpay'],
+      ['8168524506870054', 'unionpay'],
```

Comment:
Can we add a test for `8` being unionpay? Pretty sure it's the only one that can start with `8`.

Reply:
Done! Looks like you're right with your second comment about the parsing below. It was correctly identified as UnionPay.